doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -382,94 +382,64 @@ approaches that can be taken, with :numref:`Comparison of example Kubernetes net
 differences between networking solutions that consist of these options. It is important to note that different
 networking solutions require different descriptors from the Kubernetes workloads (specifically, the deployment
 artefacts such as YAML files, etc.), therefore the networking solution should be agreed between the CNF vendors and the
-CNF operators:
+CNF operators. To allow easy integration of CNF-s and the platforms it is recommended to use either the plain CNI API
+resources or the API resources defined in the v1.2 of the `Kubernetes Network Custom Resource Definition De-facto
+Standard <https://github.com/k8snetworkplumbingwg/multi-net-spec/tree/master/v1.2>`__.
 
 -  The **Default CNI Plugin** through the use of deployment specific configuration (e.g. `Tungsten Fabric
    <https://tungstenfabric.github.io/website/Tungsten-Fabric-Architecture.html#vrouter-deployment-options>`__)
 -  A **multiplexer/meta-plugin** that integrates with the Kubernetes control plane
    via CNI (Container Network Interface) and allows for use of multiple CNI plugins
    in order to provide this specific connectivity that the default Network Plugin may
-   not be able to provide (e.g. `Multus <https://github.com/intel/multus-cni>`__,
-   `DANM <https://github.com/nokia/danm>`__)
--  An external, **federated networking manager** that uses the Kubernetes API Server
-   to create and manage additional connections for Pods (e.g. `Network Service
-   Mesh <https://networkservicemesh.io>`__)
+   not be able to provide (e.g. `Multus <https://github.com/intel/multus-cni>`__)
```

Comment:
Multus is an example of the multiplexer/meta-plugin and not the default network CNI plugin
```suggestion
   not be able to provide.
```